### PR TITLE
Add basic devcontainer.

### PR DESCRIPTION
**Pull request by tinydogio-joshua**
Code to add a basic devcontainer.

I left the configuration pretty light. I've also left the `.vscode` directory in place for those that don't want (or can't) use the container. We'll need to keep them in sync.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,3 +249,23 @@ export async function DELETE() {
...
+
+## Development Setup
+
+Looking to get involved? You can begin by cloning the project and using one of the following setup options.
+
+### Installation (devcontainer)
+
+> **BETA**: We are working through using a devcontainer for development. If you have [Docker](https://docker.com) on your machine and would like to give it a go and give us some feedback, please clone the repo, and then follow the instructions on the following page in place of the Installation instructions below.
+>
+> - [Guide: Using a devcontainer](https://github.com/WH-HC-Dev/front-end/wiki/Guide:-Using-a-devcontainer)
```
Comment (pmcelhaney) [inline README.md]: This isn't visible to the public.

File: README.md [hunk 2]
```diff
@@ -249,3 +249,23 @@ export async function DELETE() {
...
+
+### Installation (Original Method)
+
+Installation will set up Git hooks and install all of the Node packages. If you're only working on Mobile code this isn't strictly necessary.
```
Comment (pmcelhaney) [inline README.md]:
```suggestion
Installation will set up Git hooks and install all of the Node packages. 
```

**Comment by pmcelhaney** [comment]
Thanks! The dev container works great. I'm going to use it going forward.

I think we should put notes on contributing in a CONTRIBUTING.md file so as not to confuse people who are only interested in _using_ Counterfact.

**Review by pmcelhaney — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tinydogio-joshua) [reply README.md]: lol. I'll fix that. Grabbed my default text and didn't update. 😀

File: README.md (hunk 1 above)
Comment (tinydogio-joshua) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (tinydogio-joshua) [reply README.md]: Fixed.

**Comment by tinydogio-joshua** [comment]
> 

I'd be interested in helping out with that when you're ready.

**Comment by pmcelhaney** [comment]
Sure, go ahead and create a CONTRIBUTING.md file. It doesn't have to be perfect. Just keep the audience in mind (this is a public repo).

**Review by pmcelhaney — APPROVED** [review]

**Pull request merged**